src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -680,6 +680,13 @@ public string IronPythonResolveTargetVersion
         public List<string> ReadNotificationIds { get; set; }
         #endregion
 
+        #region Dynamo Player and Generative Design settings
+
+        // Collections of folders used by Dynamo Player or Generative Design as entry points.
+        public Dictionary<string, List<DynamoPlayerFolderItem>> DynamoPlayerFolderItems;
```

Comment:
please add some tests that deserialize / serialize these - there are examples if you search for preference setting tests.

Reply:
Done :)